### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by ayanu**
Updated sites in the UKV domain. Several changes
Edit SSL coordinates
Added Zugspitze 
Please check Jungfraujoch and Zugspitze for model release height. I don't use the height and height_name  in the case of mountain stations.

### CODE REVIEW

**Comment by ag12733** [comment]
@ayanu - fine to add ZSF.  The SSL coordinates currently match what is in ICOS and per our conversation, we keep to the ICOS files.

**Comment by ag12733** [comment]
@ayanu  - for JFJ - NAME is released at 1000 magl and it looks like you have changed to 5?  This is inconsistent with our runs.  Need to check with Alistair

**Comment by ag12733** [comment]
@ayanu  - please remove all changes to UKV (UK and Irish sites).  I will update those in this branch directly to match what is in Dickon's files and what will match the UK releases.

File: openghg_defs/data/site_info.json [hunk 1]
```diff
@@ -396,17 +396,17 @@
       "height": ["248m", "108m", "42m"],
       "height_name": ["248magl", "110magl", "50magl"],
       "height_station_masl": 382,
-      "latitude": 54.35846,
+      "latitude": 54.3586,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:399]:
```suggestion
      "latitude": 54.35846,
```

File: openghg_defs/data/site_info.json [hunk 2]
```diff
@@ -396,17 +396,17 @@
...
       "long_name": "Bilsdale, UK",
-      "longitude": -1.15018
+      "longitude": -1.1504
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:401]:
```suggestion
      "longitude": -1.15018
```

File: openghg_defs/data/site_info.json [hunk 3]
```diff
@@ -396,17 +396,17 @@
...
     },
      "NOAA": {
       "height": ["108m", "42m", "248m"],
       "height_name": ["110magl", "50magl", "248magl"],
       "height_station_masl": 382,
-      "latitude": 54.35846,
+      "latitude": 54.3586,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:407]:
```suggestion
      "latitude": 54.35846,
```

File: openghg_defs/data/site_info.json [hunk 4]
```diff
@@ -396,17 +396,17 @@
...
       "long_name": "Bilsdale, UK",
-      "longitude": -1.15018
+      "longitude": -1.1504
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:409]:
```suggestion
      "longitude": -1.15018
```

File: openghg_defs/data/site_info.json [hunk 5]
```diff
@@ -474,10 +474,10 @@
     "ECN": {
       "height": ["120m", "20m", "60m", "200m"],
       "height_name": ["120magl", "20magl", "60magl", "200magl"],
-      "height_station_masl": 213.0,
+      "height_station_masl": 0.0,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:477]:
```suggestion
      "height_station_masl": 213.0,
```

File: openghg_defs/data/site_info.json [hunk 6]
```diff
@@ -1598,7 +1599,7 @@
     "NOAA": {
       "latitude": 46.54753,
       "longitude": 7.98498,
-      "height_station_masl": 3528,
+      "height_station_masl": 3580,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1602]:
```suggestion
      "height_station_masl": 3528,
```

File: openghg_defs/data/site_info.json [hunk 7]
```diff
@@ -681,13 +681,14 @@
       "height_name": ["6magl"]
     }
   },
-  "CPI": {
+  "CSP": {
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:684]:
```suggestion
  "CPI": {
```

File: openghg_defs/data/site_info.json [hunk 8]
```diff
@@ -681,13 +681,14 @@
...
     "ICOS": {
-      "latitude": 52.17,
-      "longitude": -6.35,
+      "latitude": 52.1775,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:686]:
```suggestion
      "latitude": 52.17,
```

File: openghg_defs/data/site_info.json [hunk 9]
```diff
@@ -681,13 +681,14 @@
...
+      "longitude": -6.3677,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:687]:
```suggestion
      "longitude": -6.35,
```

File: openghg_defs/data/site_info.json [hunk 10]
```diff
@@ -681,13 +681,14 @@
...
       "height_station_masl": 9.0,
-      "long_name": "Carnsore Point, Ireland",
-      "icos_sitecode": "CSP"
+      "height": ["10m"],
+      "height_name": ["10magl"],
+      "long_name": "Carnsore Point, Ireland"
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:691]: Would be good to add `"icos_sitecode": "CSP"` back here

File: openghg_defs/data/site_info.json [hunk 11]
```diff
@@ -1282,17 +1283,17 @@
       "height": ["100m", "50m"],
       "height_name": ["100magl", "50magl"],
       "height_station_masl": 158,
-      "latitude": 50.97661,
+      "latitude": 50.9768,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1286]:
```suggestion
      "latitude": 50.97661,
```

File: openghg_defs/data/site_info.json [hunk 12]
```diff
@@ -1282,17 +1283,17 @@
...
       "long_name": "Heathfield, UK",
-      "longitude": 0.23042
+      "longitude": 0.2305
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1288]:
```suggestion
      "longitude": 0.23042
```

File: openghg_defs/data/site_info.json [hunk 13]
```diff
@@ -1282,17 +1283,17 @@
...
     },
     "GAUGE": {
       "height": ["100m", "50m"],
       "height_name": ["100magl", "50magl"],
       "height_station_masl": 158,
-      "latitude": 50.97661,
+      "latitude": 50.9768,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1294]:
```suggestion
      "latitude": 50.97661,
```

File: openghg_defs/data/site_info.json [hunk 14]
```diff
@@ -1282,17 +1283,17 @@
...
       "long_name": "Heathfield, UK",
-      "longitude": 0.23042
+      "longitude": 0.2305
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1296]:
```suggestion
      "longitude": 0.23042
```

File: openghg_defs/data/site_info.json [hunk 15]
```diff
@@ -1944,17 +1945,17 @@
       "height": ["10m"],
       "height_name": ["10magl"],
       "height_station_masl": 8.0,
-      "latitude": 53.32663,
+      "latitude": 53.3267,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1948]:
```suggestion
      "latitude": 53.32663,
```

File: openghg_defs/data/site_info.json [hunk 16]
```diff
@@ -1944,17 +1945,17 @@
...
       "long_name": "Mace Head, Ireland",
-      "longitude": -9.90456
+      "longitude": -9.9046
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1950]:
```suggestion
      "longitude": -9.90456
```

File: openghg_defs/data/site_info.json [hunk 17]
```diff
@@ -1944,17 +1945,17 @@
...
     },
     "ICOS": {
       "height": ["10m"],
       "height_name": ["10magl"],
       "height_station_masl": 8.0,
-      "latitude": 53.32663,
+      "latitude": 53.3267,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1956]:
```suggestion
      "latitude": 53.32663,
```

File: openghg_defs/data/site_info.json [hunk 18]
```diff
@@ -1944,17 +1945,17 @@
...
       "long_name": "Mace Head, Ireland",
-      "longitude": -9.90456
+      "longitude": -9.9046
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:1958]:
```suggestion
      "longitude": -9.90456
```

File: openghg_defs/data/site_info.json [hunk 19]
```diff
@@ -2527,25 +2538,25 @@
       "height": ["90m", "45m"],
       "height_name": ["90magl", "45magl"],
       "height_station_masl": 207,
-      "latitude": 51.99738,
+      "latitude": 51.9975,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2541]:
```suggestion
      "latitude": 51.99738,
```

File: openghg_defs/data/site_info.json [hunk 20]
```diff
@@ -2527,25 +2538,25 @@
...
       "long_name": "Ridge Hill, UK",
-      "longitude": -2.53977
+      "longitude": -2.5399
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2543]:
```suggestion
      "longitude": -2.53977
```

File: openghg_defs/data/site_info.json [hunk 21]
```diff
@@ -2527,25 +2538,25 @@
...
     },
     "ICOS": {
       "height": ["90m", "45m"],
       "height_name": ["90magl", "45magl"],
       "height_station_masl": 207,
-      "latitude": 51.99738,
+      "latitude": 51.9975,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2549]:
```suggestion
      "latitude": 51.99738,
```

File: openghg_defs/data/site_info.json [hunk 22]
```diff
@@ -2527,25 +2538,25 @@
...
       "long_name": "Ridge Hill, UK",
-      "longitude": -2.53977
+      "longitude": -2.5399
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2551]:
```suggestion
      "longitude": -2.53977
```

File: openghg_defs/data/site_info.json [hunk 23]
```diff
@@ -2527,25 +2538,25 @@
...
     },
     "NOAA": {
       "height": ["90m", "45m"],
       "height_name": ["90magl", "45magl"],
       "height_station_masl": 207,
-      "latitude": 51.99738,
+      "latitude": 51.9975,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2557]:
```suggestion
      "latitude": 51.99738,
```

File: openghg_defs/data/site_info.json [hunk 24]
```diff
@@ -2527,25 +2538,25 @@
...
       "long_name": "Ridge Hill, UK",
-      "longitude": -2.53977
+      "longitude": -2.5399
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2559]:
```suggestion
      "longitude": -2.53977
```

File: openghg_defs/data/site_info.json [hunk 25]
```diff
@@ -2950,15 +2961,15 @@
       "height": ["185m", "54m", "100m"],
       "height_name": ["185magl", "50magl", "100magl"],
       "height_station_masl": 64,
-      "latitude": 52.51811,
+      "latitude": 52.5188,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2964]:
```suggestion
      "latitude": 52.51811,
```

File: openghg_defs/data/site_info.json [hunk 26]
```diff
@@ -2950,15 +2961,15 @@
...
       "long_name": "Tacolneston Tower, UK",
-      "longitude": 1.13847
+      "longitude": 1.1387
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2966]:
```suggestion
      "longitude": 1.13847
```

File: openghg_defs/data/site_info.json [hunk 27]
```diff
@@ -2950,15 +2961,15 @@
...
     },
     "NOAA": {
       "height_station_masl": 64,
       "latitude": 52.51811,
       "long_name": "Tacolneston, United Kingdom",
-      "longitude": 1.13847
+      "longitude": 1.1387
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:2972]:
```suggestion
      "longitude": 1.13847
```

File: openghg_defs/data/site_info.json [hunk 28]
```diff
@@ -3157,17 +3168,17 @@
       "height": ["222m"],
       "height_name": ["222magl"],
       "height_station_masl": 315,
-      "latitude": 56.55519,
+      "latitude": 56.5551,
       "long_name": "Angus Tower, UK",
-      "longitude": -2.98604
+      "longitude": -2.9860
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3173]:
```suggestion
      "longitude": -2.98604
```

File: openghg_defs/data/site_info.json [hunk 29]
```diff
@@ -3157,17 +3168,17 @@
...
     },
     "ICOS": {
       "height": ["222m"],
       "height_name": ["222magl"],
       "height_station_masl": 315,
-      "latitude": 56.55519,
+      "latitude": 56.5551,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3179]:
```suggestion
      "latitude": 56.55519,
```

File: openghg_defs/data/site_info.json [hunk 30]
```diff
@@ -3157,17 +3168,17 @@
...
       "long_name": "Angus Tower, UK",
-      "longitude": -2.98604
+      "longitude": -2.9860
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3181]:
```suggestion
      "longitude": -2.98604
```

File: openghg_defs/data/site_info.json [hunk 31]
```diff
@@ -3250,27 +3270,27 @@
       "height": ["10m"],
       "height_name": ["20magl"],
       "height_station_masl": 17.0,
-      "latitude": 52.95,
+      "latitude": 52.9504,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3273]:
```suggestion
      "latitude": 52.95,
```

File: openghg_defs/data/site_info.json [hunk 32]
```diff
@@ -3250,27 +3270,27 @@
...
       "long_name": "WAO",
-      "longitude": 1.121,
+      "longitude": 1.1219,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3275]:
```suggestion
      "longitude": 1.121,
```

File: openghg_defs/data/site_info.json [hunk 33]
```diff
@@ -3250,27 +3270,27 @@
...
       "reference": "https://meta.icos-cp.eu/resources/stations/AS_WAO",
       "comments": "05/01/2023: Station elevation of 31m may be included within ICOS header but value above of 17m is more recently agreed upon value and should be correct."
     },
     "UCAM": {
       "height": ["10m"],
       "height_name": ["20magl"],
       "height_station_masl": 17.0,
-      "latitude": 52.95042,
+      "latitude": 52.9504,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3283]:
```suggestion
      "latitude": 52.95042,
```

File: openghg_defs/data/site_info.json [hunk 34]
```diff
@@ -3250,27 +3270,27 @@
...
       "long_name": "Weybourne Observatory, UK",
-      "longitude": 1.12194
+      "longitude": 1.1219
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3285]:
```suggestion
      "longitude": 1.12194
```

File: openghg_defs/data/site_info.json [hunk 35]
```diff
@@ -3250,27 +3270,27 @@
...
     },
     "UEA": {
       "height": ["10m"],
       "height_name": ["20magl"],
       "height_station_masl": 17.0,
-      "latitude": 52.95042,
+      "latitude": 52.9504,
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3291]:
```suggestion
      "latitude": 52.95042,
```

File: openghg_defs/data/site_info.json [hunk 36]
```diff
@@ -3250,27 +3270,27 @@
...
       "long_name": "Weybourne Observatory, UK",
-      "longitude": 1.12194
+      "longitude": 1.1219
```
Comment (rt17603) [inline openghg_defs/data/site_info.json:3293]:
```suggestion
      "longitude": 1.12194
```

File: openghg_defs/data/site_info.json (hunk 10 above)
Comment (rt17603) [reply openghg_defs/data/site_info.json:691]: Don't need this since site name has now been updated to "CSP" to match to ICOS

**Comment by rt17603** [comment]
Overall changes - Adding new sites including Zugspitze Schneefernerhaus (ZSF), Malin Head (MLN), Valentia (VAL) and updating site code for Carnsore (to CSP)

**Pull request merged**